src/main/java/seedu/address/ui/OrderCard.java

```diff
@@ -38,28 +38,30 @@
     @FXML
     private Label warehouse;
     @FXML
-    private Label email;
+    private Label cashOnDelivery;
     @FXML
     private Label comment;
     @FXML
     private Label timeStamp;
     @FXML
-    private FlowPane tags;
+    private FlowPane itemType;
 
     public OrderCard(Order order, int displayedIndex) {
         super(FXML);
         this.order = order;
         id.setText(displayedIndex + ". ");
+        tid.setText(order.getTid().tid);
         name.setText(order.getName().fullName);
         phone.setText(order.getPhone().value);
         address.setText(order.getAddress().value);
         timeStamp.setText(order.getTimestamp().value);
         warehouse.setText(order.getWarehouse().address);
-        email.setText(order.getEmail().value);
         comment.setText(order.getComment().commentMade);
-        order.getTags().stream()
-                .sorted(Comparator.comparing(tag -> tag.tagName))
-                .forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
+        cashOnDelivery.setText(order.getCash().cashOnDelivery);
+
+        if (!(order.getItemType().itemType).equals("NIL")) {
+            itemType.getChildren().add(new Label(order.getItemType().itemType));
+        }
```

Comment:
I believe we could only have one item type per parcel, instead of having multiple parcel?
Does this portion allows the user to add multiple item type for one parcel?

You could change to Label instead of using the FlowPane just like the rest.